### PR TITLE
Ui add swap component

### LINKED ISSUE #2278: If Eth gas station is down or has not responded use reasonable gas estimate
Currently if will instead just be blank and try to use a 0 gas price.

### LINKED ISSUE #4893: Loading modal
Update copy to: 
- "Connecting to our partners at [wallet provider] to create your secure account."
- When Torus is the wallet provider copy should be 'Torus', not 'Tor.us' 

Add learn more link. 

Figma: 
https://www.figma.com/file/RMXc7jUWiC6amxz9Q2JK5L/Onboarding-and-Wallet-UX?node-id=1588%3A415

### LINKED ISSUE #5447: Background image is stretching and imagery is being warped
The background image used in the banner warps at different sizes, we need to keep that to it's original fixed ratio to avoid that

### PR DESCRIPTION
**Pull request by andrewdamelio**
- first pass at swap. Adding UI components and wiring up to AddFunds (REP flow). Middleware work still required for swap/price feeds.

- closes #5447 [Background image is stretching and imagery is being warped]
- closes #4893 [Loading modal fixes]
- closes #2278  [If Eth gas station is down or has not responded use reasonable gas estimate]

### CODE REVIEW

File: packages/augur-ui/src/modules/modal/add-funds.tsx [hunk 1]
```diff
@@ -135,7 +152,25 @@ export const AddFunds = ({
           <BackButton action={() => setSelectedOption(() => null)} />
           <CloseButton action={() => closeAction()} />
         </div>
-        <div className={selectedOption === '3' ? Styles.AddFundsTransfer : Styles.AddFundsCreditDebitCoinbase}>
+        <div
+          className={
+            selectedOption === '3'
```
Comment (bthaile) [inline packages/augur-ui/src/modules/modal/add-funds.tsx]: Can you use constants for ‘0’ and ‘3’, ... to make it more readable

Comment (andrewdamelio) [reply packages/augur-ui/src/modules/modal/add-funds.tsx]: 👍

File: packages/augur-ui/src/modules/swap/components/index.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+interface SwapProps {
+  balances: AccountBalances;
+  toToken: string;
+  fromToken: string;
+}
+
+export const Swap = ({ balances, fromToken, toToken }: SwapProps) => {
+  // TODO placeholder rates until price feed is hooked up
+  const ETH_RATE = 160.63;
+  const REP_RATE = 15.87;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/swap/components/index.tsx]: Can you put the constants in the container so its super easy to hook up

File: packages/augur-ui/src/modules/swap/components/index.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+  const setFromToken = () => {
+    const nextToken = selectedToken === ETH ? DAI : ETH;
+    setSelectedFromTokenAmount('');
+    setSelectedToken(nextToken);
+  };
+
+  const setTokenAmount = (amount, displayBalance) => {
+    if (amount < 0) {
```
Comment (bthaile) [inline packages/augur-ui/src/modules/swap/components/index.tsx]: Should be using BigNumber for everything. Tokens amounts can go out to 18 digits.

Comment (andrewdamelio) [reply packages/augur-ui/src/modules/swap/components/index.tsx]: Ah darn I meant to update that, fixing

**Comment by bthaile** [comment]
PR looks good, concerned about conversions and using big number in the swap stuff.

**Review by bthaile — APPROVED** [review]
minor nit picks, PR looks good

**Comment by joeykrug** [comment]
we need ability to add liquidity to 0x too, idk where we wanna put that @andrewdamelio

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). 🥇
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/exohmn0**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

**Comment by andrewdamelio** [comment]
@joeykrug Let me chat with @cillianhunter0  about that

**Comment by andrewdamelio** [comment]
> we need ability to add liquidity to 0x too, idk where we wanna put that @andrewdamelio

@joeykrug Did you mean add liquidity to uniswap pools? Otherwise i think i misunderstood.

**Pull request merged**